CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Team Moon-glE Contributing Guidelines
+
+Please follow these guidelines when you are trying to contribute to MCIP software.
+
+## Language
+
+We **_prefer English and Korean_** due to all of our team member is Korean, we prefer Korean. So it is strongly recommended to use Korean. But it needs to conversate in gobally, we also prefer the English.
+
+## Code Formatting
+
+Keep your codes clean and consistent using code formatters and linters. Most of the time formatting and linting would already have been set by default in each repository. And sometimes you should use the specific code editor to match the configuration.
```

Comment:
`Keep your codes clean and consistent using code formatters and linters. Most of the time formatting and linting would already have been set by default in each repository. And sometimes you should use the specific code editor to match the configuration.` 
Is too generalized.

`Keep your codes clean and consistent using code formatters and linters. we prefer to use basic PEP8 format.` seems to be appropriate for our project